fabric-network/test/impl/event/contractlistener.spec.ts

```diff
@@ -5,12 +5,12 @@
  */
 
 import sinon = require('sinon');
-import chai = require('chai');
-const expect = chai.expect;
+import 'mocha';
```

Comment:
This doesn't look correct

Reply:
removed, it was used to allow for debug under vscode